test/types/string/dmk/utf8_values.chpl

```diff
@@ -0,0 +1,9 @@
+var mystring1 = "événement";
+
+for c in mystring1.uchars() {
+  const str = codePointToString(c);
+  write(c, "\t", str);
+  for i in 0..#str.length do
+    write(" ", str.buff[i]:int);
+  writeln();
+}
```

Comment:
You definitely need more testing - especially strip testing left and right sides. Ideally every string fn with unicode support should have a test.